kitchen-sink/react/pages/Notification.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react';
+import {
+  Page,
+  Navbar,
+  NavbarBackLink,
+  Block,
+  Notification,
+  Button,
+  Dialog,
+  DialogButton,
+  useTheme,
+} from 'konsta/react';
+import { XmarkCircleFill } from 'framework7-icons/react';
+import { MdOutlineCancel } from 'react-icons/md';
+import DemoIcon from '../components/DemoIcon';
+
+export default function NotificationPage() {
+  const isPreview = document.location.href.includes('examplePreview');
+  const [notificationFull, setNotificationFull] = useState(false);
+  const [notificationWithButton, setNotificationWithButton] = useState(false);
+  const [notificationCloseOnClick, setNotificationCloseOnClick] =
+    useState(false);
+  const [notificationCallbackOnClose, setNotificationCallbackOnClose] =
+    useState(false);
+  const [alertOpened, setAlertOpened] = useState(false);
+  const theme = useTheme();
+  const CloseIcon = theme === 'ios' ? XmarkCircleFill : MdOutlineCancel;
+
+  const openNotification = (setter) => {
+    setNotificationFull(false);
```

Comment:
How to close this notification? I think it should be closed automatically within some time interval

Reply:
Same for Vue and Svelte